### DISCUSSION TITLE
NBODY in 2nd order IFCs

### DISCUSSION THREAD

**Discussion by RicardoCecconello** [ttadano/alamode#179] [Q&A] [answered]
Dear Dr. Tadano,

Why is not possible to include 3-body interactions in the harmonic part of the IFCs? For example, a 3-body interaction (involving an angular term/variation) could still be modeled with 2nd order IFCs. Besides, we can include 4-body interactions in the 3rd order IFCs.

Thanks in advance,
Ricardo.

**Comment by ttadano** [comment, marked as answer]
The $n$ th order force constant is defined as

```math
\Phi_{\mu_1\dots \mu_n} (\ell_1\kappa_1;\dots;\ell_n\kappa_n) = \frac{\partial^{n} U}{\partial u_{\mu_1}(\ell_1\kappa_1)\cdots \partial u_{\mu_n}(\ell_n\kappa_n)}
```
where $u_{\mu}(\ell\kappa)$ is the atomic displacement and $U$ is the Born-Oppenheimer potentail energy surface. https://alamode.readthedocs.io/en/latest/almdir/formalism_alm.html#interatomic-force-constants-ifcs

With this definition, the second-order IFCs involve only two atom indices, $(\ell_1\kappa1, \ell_2\kappa_2)$, so there are no multibody (3-, 4-body, etc.) interactions.
